lib/bottom_navy_bar.dart

```diff
@@ -167,7 +206,7 @@ class _ItemWidget extends StatelessWidget {
                 if (isSelected)
                   Expanded(
                     child: Container(
-                      padding: EdgeInsets.symmetric(horizontal: 4),
+                      padding: itemPadding,
```

Comment:
We need to make sure to not break any UI with those changes. So far, removing the horizontal padding may break some existing users

Reply:
I'll change the defaults for itemPadding to be EdgeInsets.symmetric(horizontal: 4), that way it won't break anyone's apps